### PR TITLE
Add static tagging system.

### PR DESCRIPTION
**Pull request by tommyip**
In the future, clicking on a tag should bring the user to a page showing all the posts with the same tags.

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -14,3 +14,33 @@ a:hover {
...
+    margin: 0;
+    overflow: hidden;
+    padding: 0;
+}
+
+.tags li {
+    float: left;
+}
+
+.tag {
```
Comment (Arthelon) [inline css/style.css:29]: add a "cursor: pointer;" style around here. Just a small annoyance :)

File: css/style.css [hunk 2]
```diff
@@ -14,3 +14,33 @@ a:hover {
...
+    background: #eee;
+    border-radius: 3px;
+    color: #777;
```
Comment (Arthelon) [inline css/style.css]: Let's try to keep the colors consistent here. Maybe use #159957, the color used for the titles, or anything else that you think would fit well.

**Review by Arthelon — DISMISSED** [review]
Minor styling touches aside, the hover change indicates further action. If we want to keep this in we should look towards adding a page containing posts of certain tags (any tag filtering method really).

File: css/style.css [hunk 3]
```diff
@@ -14,3 +14,33 @@ a:hover {
...
+
+.tags {
+    list-style: none;
+    margin: 0;
+    overflow: hidden;
+    padding: 0;
+}
+
+.tags li {
+    float: left;
```
Comment (collinsnji) [inline css/style.css:26]: Use `float` with care. Or use a `flex` display if possible

**Comment by tommyip** [comment]
Yeah I will open an issue for the page containing filtered page.

File: css/style.css (hunk 3 above)
Comment (tommyip) [reply css/style.css:26]: What's wrong with float?

File: css/style.css (hunk 2 above)
Comment (tommyip) [reply css/style.css]: Keeping it grey looks better here.

**Pull request merged**